plugins/qcheck-stm/src/stm_of_ir.ml

```diff
@@ -431,7 +455,25 @@
     subst_term state ~gos_t:id ~old_t:None ~new_t:(Some new_state_ident)
       ~new_lz:true t.term
     >>= ocaml_of_term config
-  and wrap_check t e =
+  and dummy =
+    let ty_show = qualify [ "Ortac_runtime" ] "dummy" and value = eunit in
+    let args = pexp_tuple_opt [ ty_show; value ] in
+    pexp_construct res args
+  in
+  let* ret_val =
+    (* simply warn the user if we can't compute the expected returned value,
+       don't skip the function *)
+    match returned_value translate_postcond value with
+    | None ->
+        let* () =
+          warn
+            ( Incomplete_ret_val_computation (Fmt.str "%a" Ident.pp value.id),
+              value.id.id_loc )
+        in
+        ok @@ esome dummy
+    | Some e -> ok @@ esome e
+  in
+  let wrap_check ?(normal = false) t e =
```

Comment:
What does `normal` really mean here? I see it set to `true` for (normal) postconditions and invariants, but anyway, I don’t have a clear understanding of the impact of setting it to `true`.

Reply:
I've added some comments to explain.
Thanks!